MusicBot Error I want to know how to fix

Please tick all applicable boxes.

- [/ ] I am using Python 3.5.3 or higher (run `python --version` on the command line)
- [ /] I have followed the [official guides](https://github.com/Just-Some-Bots/MusicBot/wiki) to install the bot for my system
- [ /] I have updated my dependencies to the latest version using the appropriate `update` script

Which version are you using?

- [ /] The latest `master` version (release-260819)
- [ ] The latest `review` version

What type of issue are you creating?

- [ ] Bug
- [ ] Feature request
- [ /] Question

----

### Description of issue



### Steps to reproduce

1. 

----

### Log file
Please attach your MusicBot log file (located at `logs/musicbot.log`) to this issue. You can do so by dragging and dropping the file here. **If you do not include your log file, you WILL b
[musicbot.log](https://github.com/Just-Some-Bots/MusicBot/files/4967254/musicbot.log)
e asked to provide one.**

**Before, I was able to use bots without problems. But today, there is a problem as listed below.**

[INFO] launcher: Starting sanity checks
[INFO] launcher: Checking for Python 3.5+
[INFO] launcher: Checking console encoding
[INFO] launcher: Setting console encoding to UTF-8
[INFO] launcher: Ensuring we're in the right environment
[INFO] launcher: Adding local bins/ folder to path
[INFO] launcher: Required checks passed.
[INFO] launcher: Optional checks passed.
[INFO] launcher: Moving old musicbot log

Starting MusicBot release-260819-5-gc87dc05
Loaded autoplaylist with 67 entries
[ERROR] launcher: Error starting bot
Traceback (most recent call last):
  File "run.py", line 373, in main
    m.run()
  File "C:\Users\AzIToB\Desktop\MusicBot\musicbot\bot.py", line 902, in run
    self.loop.run_until_complete(self.start(*self.config.auth))
  File "C:\Users\AzIToB\AppData\Local\Programs\Python\Python37-32\lib\asyncio\base_events.py", line 568, in run_until_complete
    return future.result()
  File "C:\Users\AzIToB\AppData\Roaming\Python\Python37\site-packages\discord\client.py", line 543, in start
    await self.connect(reconnect=reconnect)
  File "C:\Users\AzIToB\AppData\Roaming\Python\Python37\site-packages\discord\client.py", line 457, in connect
    await self._connect()
  File "C:\Users\AzIToB\AppData\Roaming\Python\Python37\site-packages\discord\client.py", line 421, in _connect
    await self.ws.poll_event()
  File "C:\Users\AzIToB\AppData\Roaming\Python\Python37\site-packages\discord\gateway.py", line 469, in poll_event
    await self.received_message(msg)
  File "C:\Users\AzIToB\AppData\Roaming\Python\Python37\site-packages\discord\gateway.py", line 423, in received_message
    func(data)
  File "C:\Users\AzIToB\AppData\Roaming\Python\Python37\site-packages\discord\state.py", line 663, in parse_guild_create
    guild = self._get_create_guild(data)
  File "C:\Users\AzIToB\AppData\Roaming\Python\Python37\site-packages\discord\state.py", line 638, in _get_create_guild
    guild._from_data(data)
  File "C:\Users\AzIToB\AppData\Roaming\Python\Python37\site-packages\discord\guild.py", line 391, in _from_data
    self._sync(guild)
  File "C:\Users\AzIToB\AppData\Roaming\Python\Python37\site-packages\discord\guild.py", line 420, in _sync
    self._add_channel(VoiceChannel(guild=self, data=c, state=self._state))
  File "C:\Users\AzIToB\AppData\Roaming\Python\Python37\site-packages\discord\channel.py", line 504, in __init__
    self._update(guild, data)
  File "C:\Users\AzIToB\AppData\Roaming\Python\Python37\site-packages\discord\channel.py", line 535, in _update
    self._fill_overwrites(data)
  File "C:\Users\AzIToB\AppData\Roaming\Python\Python37\site-packages\discord\abc.py", line 271, in _fill_overwrites
    self._overwrites.append(_Overwrites(id=overridden_id, **overridden))
TypeError: __new__() got an unexpected keyword argument 'deny_new'

[INFO] launcher: All done.

C:\Users\AzIToB\Desktop\MusicBot>

//Thank you for all your help.//

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,5 +1,5 @@
-pynacl==1.4.0
-discord.py[voice]==1.3.4
+pynacl
+discord.py[voice]
 pip
 youtube_dl
 colorlog

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,5 +1,5 @@
-pynacl==1.3.0
-discord.py[voice]==1.2.5
+pynacl==1.4.0
+discord.py[voice]==1.3.4
 pip
 youtube_dl
 colorlog

